src/transformers/__init__.py

```diff
@@ -214,6 +214,7 @@
     from .models.camembert import CamembertTokenizer
     from .models.marian import MarianTokenizer
     from .models.mbart import MBartTokenizer
+    from .models.mt5 import MT5Tokenizer
```

Comment:
MT5 just uses T5's tokenizer, so there is no `tokenization_mt5.py` file. Not sure I fully understand what's going on here